### PR TITLE
fix(openebs-install): check node details of the cluster

### PR DESCRIPTION
**Pull request by harshshekhar15**
Signed-off-by: Harsh Shekhar <harshshekhar15@gmail.com>

This PR intends to do the following:
- Check node details of the cluster on which we want to install openebs. If the node details is empty retry until it gets updated with all the details.
- Fix label cleanup task.

#### Versions:

```
$ Kubernetes version
$ Kubernetes platform
$ kubectl version

```

### CODE REVIEW

File: litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml [hunk 1]
```diff
@@ -72,6 +73,9 @@
             body_format: json
             status_code: 200
           register: node_details
+          until: "node_details.json.data != []"
+          delay: 2
+          retries: 10
```
Comment (gprasath) [inline litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml:78]: is this sufficient?

Comment (harshshekhar15) [reply litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml:78]: I guess so @gprasath. Do you have any thoughts?

Comment (gprasath) [reply litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml:78]: Keep the delay as it is and increase the retries

Comment (harshshekhar15) [reply litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-CP-ON-SPECIFIC-NODE/test.yml:78]: Ok @gprasath. I'll increase the reries to 30. That should be enough I guess.

**Pull request merged**